generator/statistics.cpp

```diff
@@ -55,20 +60,50 @@ namespace stats
     {
       f.ParseBeforeStatistic();
 
-      FeatureType::InnerGeomStat const innerStats = f.GetInnerStatistic();
+      FeatureType::InnerGeomStat const innerStats = f.GetInnerStats();
 
       m_info.m_inner[0].Add(innerStats.m_points);
       m_info.m_inner[1].Add(innerStats.m_strips);
       m_info.m_inner[2].Add(innerStats.m_size);
 
-      // get geometry size for the best geometry
-      FeatureType::GeomStat const geom = f.GetGeometrySize(FeatureType::BEST_GEOMETRY);
-      FeatureType::GeomStat const trg = f.GetTrianglesSize(FeatureType::BEST_GEOMETRY);
+      // Get size stats and load the best geometry.
+      FeatureType::GeomStat const geom = f.GetOuterGeometryStats();
+      FeatureType::GeomStat const trg = f.GetOuterTrianglesStats();
+
+      uint32_t geomSize = 0, trgSize = 0;
+      int const n = feature::DataHeader::kMaxScalesCount;
```

Comment:
Please use kMaxScalesCount directly for better readability of the code below. You can also try `using` for a less verbose syntax.